ktor-client/ktor-client-tests/jvm/src/io/ktor/client/tests/utils/tests/Headers.kt

```diff
@@ -17,7 +18,16 @@ internal fun Application.headersTestServer() {
                 call.response.header("X-Header-Single-Value", "foo")
                 call.response.header("X-Header-Double-Value", "foo")
                 call.response.header("X-Header-Double-Value", "bar")
-                call.respond(HttpStatusCode.OK, "OK")
+                val message = call.request.header(HttpHeaders.ContentLength) ?: ""
```

Comment:
Minor: maybe create a special route for `content-length` header and not modify this one? But up to you.